core/src/klio_core/config/_io.py

```diff
@@ -286,6 +286,12 @@ def as_dict(self):
 @attr.attrs(frozen=True)
 class KlioWriteFileConfig(KlioEventOutput, KlioFileConfig):
     file_path_prefix = attr.attrib(type=str)
+    file_name_suffix = attr.attrib(type=str, default="")
+    append_trailing_newlines = attr.attrib(type=bool, default=True)
+    num_shards = attr.attrib(type=int, default=0)
+    shard_name_template = attr.attrib(type=str, default=None)
+    compression_type = attr.attrib(type=str, default="auto")
```

Comment:
Out of curiosity, is `"auto"` == to `CompressionTypes.AUTO` (the [default in the `WriteToText`](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/io/textio.py#L595) class that `KlioWriteToText` wraps)? Just want to make sure they match in equality; or rather, that when provided, Beam knows what `"auto"` means. (do I make sense? 😬 )